apps/journeys-admin/pages/journeys/[journeyId]/reports/visitors.tsx

```diff
@@ -19,16 +20,113 @@ import i18nConfig from '../../../../next-i18next.config'
 import { createApolloClient } from '../../../../src/libs/apolloClient'
 import { ACCEPT_USER_INVITE } from '../../..'
 import { UserJourneyOpen } from '../../../../__generated__/UserJourneyOpen'
+import { JourneyVisitorsList } from '../../../../src/components/JourneyVisitorsList'
+import {
+  GetJourneyVisitors,
+  GetJourneyVisitors_visitors_edges as VisitorEdge
+} from '../../../../__generated__/GetJourneyVisitors'
+import { GetJourneyVisitorsCount } from '../../../../__generated__/GetJourneyVisitorsCount'
+
+export const GET_JOURNEY_VISITORS = gql`
+  query GetJourneyVisitors(
+    $filter: JourneyVisitorFilter!
+    $first: Int
+    $after: String
+  ) {
+    visitors: journeyVisitorsConnection(
+      teamId: "jfp-team"
+      filter: $filter
+      first: $first
+      after: $after
+    ) {
+      edges {
+        cursor
+        node {
+          visitorId
+          countryCode
+          createdAt
+          duration
+          visitor {
+            name
+            status
+            referrer
+          }
+          events {
+            id
+            createdAt
+            label
+            value
+          }
+        }
+      }
+      pageInfo {
+        hasNextPage
+        startCursor
+        endCursor
+      }
+    }
+  }
+`
+
+export const GET_JOURNEY_VISITORS_COUNT = gql`
+  query GetJourneyVisitorsCount($filter: JourneyVisitorFilter!) {
+    journeyVisitorCount(filter: $filter)
+  }
+`
 
 function JourneyVisitorsPage(): ReactElement {
   const { t } = useTranslation('apps-journeys-admin')
   const AuthUser = useAuthUser()
   const router = useRouter()
-
   const journeyId = router.query.journeyId as string
 
   useTermsRedirect()
 
+  const { data } = useQuery<GetJourneyVisitorsCount>(
+    GET_JOURNEY_VISITORS_COUNT,
+    {
+      variables: {
+        filter: { journeyId }
+      }
+    }
+  )
+
+  const [visitorEdges, setVisitorEdges] = useState<VisitorEdge[]>([])
+  const [hasNextPage, setHasNextPage] = useState(false)
+  const [endCursor, setEndCursor] = useState<string | null>()
+
+  const { fetchMore, loading } = useQuery<GetJourneyVisitors>(
+    GET_JOURNEY_VISITORS,
+    {
+      variables: {
+        filter: { journeyId },
+        first: 50
```

Comment:
Steven has asked for this to be 100
```suggestion
        first: 100
```